lib/src/native/controller/controller.dart

```diff
@@ -377,4 +377,41 @@ class NativeAdController extends LoadShowAd<NativeAdEvent>
       assert(!reason.isNegative, 'You must specify a valid reason');
     return channel.invokeMethod('muteAd', {'reason': reason});
   }
+
+  Future<String?> getHeadline() async {
```

Comment:
It'd be better to use getters instead of `getHeadline`. That would be `Future<String?> get headline async {...}`